tools/cold-backup/README.md

```diff
@@ -22,36 +25,35 @@ Consumer group offsets are included, but not Kafka Connect, MirrorMaker and Kafk
 - enough disk space
 
 ## Test procedure
+
 ```sh
+CLIENT_IMAGE="quay.io/strimzi/kafka:latest-kafka-2.8.0"
+krun() { kubectl run client -it --rm=true --restart=Never --image=$CLIENT_IMAGE -- $@; }
```

Comment:
I'm not sure I like this. This probably makes it much harder to understand what the test procedure does. I also don't understand why the image name should be defined in environment variable instead of directly in the field. `krun` is AFAIK also an alias many people already have for `kubectl run`, so it might conflict.

Reply:
Ok. I can revert that.